chess-game/src/components/piece/basicPieces.jsx

```diff
@@ -15,7 +15,7 @@ const Pieces = () => {
       let x = index * width/array.length + spaceWidth/2;
       function movement(e) {
         x = x + 75; 
-        console.log('hhello')
+        console.log('hello')
         return x
       }
       items.push(      
```

Comment:
I'm seeing an error in this loop. Put the key on the wrapper element, not the children:
`<g key={index}>`
You can remove the keys from Piece.

Reply:
Problem fixed, repushed